### DISCUSSION TITLE
Redux or Context API?

### DISCUSSION THREAD

**Discussion by rafahcerqueira** [community/community#54305] [Programming Help] [answered]
### Body

What is the best ReactJS state management between Redux and Context API, can someone help me with this question?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by DanielAlmeidaToledo** [comment, marked as answer]
It depends a lot on the size of your project, if it's a small project I recommend the context api even for its simplicity. But for larger projects Redux behaves better.
